app/Http/Controllers/VotesController.php

```diff
@@ -204,4 +205,31 @@ public function removeVote(Request $request, $id)
         $this->destroyVote($id);
         return AdministrationController::votes();
     }
-}
+
+    public function checkFakeEmails()
+    {
+        $api_key = env('API_KEY_EMAIL_VERIFICATION');
+        $api_url = 'https://emailverification.whoisxmlapi.com/api/v2';
+
+        $votes = Vote::all();
+
+        foreach ($votes as $vote) {
+            $response = Http::get($api_url, [
+                'apiKey' => $api_key,
+                'emailAddress' => $vote['voter_email'],
+                'format' => 'json'
+            ]);
+
+            if ($response->ok()) {
+                $data = $response->json();
+                $vote->isFake = $data['smtpCheck'];
```

Comment:
! added

Reply:
![image](https://user-images.githubusercontent.com/49519258/228542024-eb83116b-1123-4818-8395-07960d9e90f5.png)
Potential issue I dont know how to avoid